routes/api.js

```diff
@@ -331,7 +331,7 @@ const approveAccount = async (req, email) => {
   const mailToken = jwt.sign({
     type: 'create_account',
     email,
-  }, process.env.JWT_SECRET, { expiresIn: '7d' });
```

Comment:
Have you verified that `expiresIn` does not have a default value that will be used if not set explicitly?

Reply:
Here 2 examples:
What was done before:
eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0eXBlIjoiY3JlYXRlX2FjY291bnQiLCJlbWFpbCI6ImVtYWlsQGVtYWlsLmNvbSIsImlhdCI6MTUyMTEwMjA5NCwiZXhwIjoxNTIxNzA2ODk0fQ.2Hso1K3_4GhnL196b-ho2diZioqjiDDLQYlzUYP1q4E

```
{
  "type": "create_account",
  "email": "email@email.com",
  "iat": 1521102094,
  "exp": 1521706894
}
```

With the fix
eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0eXBlIjoiY3JlYXRlX2FjY291bnQiLCJlbWFpbCI6ImVtYWlsQGVtYWlsLmNvbSIsImlhdCI6MTUyMTEwMjA5NH0.SGkxuGM2VwSPF8v7_BcWNnScxoegxTH27NwkwXIqLaQ

```
{
  "type": "create_account",
  "email": "email@email.com",
  "iat": 1521102094
}
```

There is no more expiry

Reply:
Ok! Thanks for verifying that, should be good to merge then